tst.py

```diff
@@ -0,0 +1,43 @@
+from TM1py.Services import TM1Service
```

Comment:
We will need to create a dedicated test module for Sandboxes in line with the test modules for chores, cubes, etc. Then we can remove this prototype file.

I suppose we can do the tests in a separate MR since this MR is already quite big.